src/components/CommitChartLabel/index.tsx

```diff
@@ -78,7 +79,7 @@ const LabelContainer = styled.div`
 const LabelText = styled(Body)`
   color: ${gray.dark2};
   width: 100%;
-  font-size: 12px;
```

Comment:
Sorry for being picky but can we change this back to `12px`? It's okay if it doesn't have a token, I just think it looks nicer 😔